ts/parts/Navigator.ts

```diff
@@ -56,7 +56,6 @@ declare global {
             navigator?: NavigatorOptions;
         }
         interface Series {
-            baseSeries?: Series;
```

Comment:
What is the reason behind the removal of the `baseSeries` property?

Reply:
It is actually part of the mixins DerivedSeries class and is only used in the Navigator.

Reply:
Conclusion after internal discussion with @bre1470, @oysteinmoseng, and @jon-a-nygaard is that the `baseSeries` property in Navigator is a different mechanism than the one in DerivedSeries, which means that we would like to still keep it in Navigator.ts.